scripts/Package.swift

```diff
@@ -0,0 +1,7 @@
+// swift-tools-version:5.5
+
+import PackageDescription
```

Comment:
why does the scripts folder have Package.swift?

Reply:
same as above